website/docs/r/sm_private_certificate_configuration_action_set_signed.html.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_sm_private_certificate_configuration_action_set_signed"
+description: |-
+Manages PrivateCertificateConfigurationActionSetSigned.
```

Comment:
I think a tab space is missing for `Manages PrivateCertificateConfigurationActionSetSigned.`
If the tab is not maintained it will not fall under [Secrets Manager](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/sm_arbitrary_secret) section in registry docs.
Kindly check other secrets manager docs and add the identation accordingly. 

This applies to csr doc as well.